README.md

```diff
@@ -1 +1,22 @@
 # AllegroPlanet
+
+Setup guide for developers
```

Comment:
Could we change this subtitle to something that includes "for Unix"?  Something like `Setup Guide for Unix` would be good.  😊